src/auth_client_demo_assets/svelte/src/routes/+page.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+  import LoggedIn from "../components/LoggedIn.svelte";
+  import LoggedOut from "../components/LoggedOut.svelte";
+  import { auth } from "../stores/auth";
+  import { onMount } from "svelte";
+
+  onMount(() => {
+    window.global = window;
+    $auth.init();
```

Comment:
this function isn't async?

Reply:
Since I'm really only waiting for the `isReady` state change, I don't really need to block waiting for init to complete. I can just fire it and let the component update as the info comes in

Reply:
Sounds good, makes sense